### PR TITLE
Update dependencies to current versions

### PR DESCRIPTION
**Pull request by ennru**
References https://github.com/akka/alpakka-kafka/issues/1191

### CODE REVIEW

File: alpakka-sample-file-to-elasticsearch/project/Dependencies.scala [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import sbt._
 
 object Dependencies {
-  val scalaVer = "2.13.2"
+  val scalaVer = "2.13.3"
   // #deps
-  val AkkaVersion = "2.6.4"
-  val AlpakkaVersion = "2.0.0"
+  val AkkaVersion = "2.6.8"
```
Comment (patriknw) [inline alpakka-sample-file-to-elasticsearch/project/Dependencies.scala:6]: 2.6.9 while you are at it?

**Review by patriknw — COMMENTED** [review]
maven and gradle?

File: alpakka-sample-sqs-java/build.sbt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-ThisBuild / scalaVersion := "2.13.2"
+ThisBuild / scalaVersion := "2.13.3"
 
-val akkaVersion = "2.6.4"
-val alpakkaVersion = "2.0.0"
-val jacksonVersion = "2.10.3"
+val AkkaVersion = "2.6.9"
+val alpakkaVersion = "2.0.1"
+val jacksonVersion = "2.10.5"
```
Comment (seglo) [inline alpakka-sample-sqs-java/build.sbt:8]: [minor] Different convention?

**Review by seglo — APPROVED** [review]
LGTM

File: alpakka-sample-sqs-java/build.sbt (hunk 2 above)
Comment (ennru) [reply alpakka-sample-sqs-java/build.sbt]: Thanks, it was even broken https://travis-ci.com/github/akka/alpakka-samples/jobs/383124399#L211

Comment (seglo) [reply alpakka-sample-sqs-java/build.sbt]: I was a compiler in another life.

**Pull request merged**